frontend/src/stimulus/controllers/pattern-input.controller.ts

```diff
@@ -444,7 +444,7 @@ export default class PatternInputController extends Controller {
         key,
         label: group.title,
         values: group.tokens
-          .filter((token) => token.key.includes(word) || token.label.includes(word) || word === '*')
+          .filter((token) => token.key.includes(word) || token.label.toLowerCase().includes(word) || word === '*')
```

Comment:
Do we know that `word` is already lower-cased? Otherwise it seems to me that searching for `Assignee` would never work.

Reply:
the word is always lower case, it is converted when parsed from the caret position.

Reply:
:green_circle: I think preferably we'd do the lower-casing of `word` inside `getFilteredSuggestionsData`, so that this method is responsible for ensuring matching characters alone and isn't dependent on callers knowing how to pass in a word.

Reply:
true enough, with a near future patch I will move that.